cmake/Config.cmake.in

```diff
@@ -1,2 +1,7 @@
 @PACKAGE_INIT@
+
+if ( CMAKE_HOST_UNIX AND NOT (MINGW OR CYGWIN) ) 
```

Comment:
`find_package(Threads REQUIRED)` should work on all platfroms unconditionally as far as I know.

Reply:
Threads on Windows is included in Kernel32.dll, IIRC.  I seem to remember spending a bit of time of this, but I will revisit.